packages/editor/src/client/i18n.ts

```diff
@@ -1,9 +1,6 @@
 import i18n from 'i18next'
 import {initReactI18next} from 'react-i18next'
 import LanguageDetector from 'i18next-browser-languagedetector'
-import {registerLocale, setDefaultLocale} from 'react-datepicker'
-
-import dateFnsDe from 'date-fns/locale/de'
```

Comment:
I'm confused don't you need this.

Reply:
@roosnic1 i think this one is outdated, I can't find it in the `files changed`

Reply:
@roosnic1 also from merge

Reply:
Same here